receiver/awscontainerinsightreceiver/README.md

```diff
@@ -1,8 +1,14 @@
 # AWS Container Insights Receiver
 
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [beta]        |
+| Supported pipeline types | logs, metrics |
```

Comment:
```suggestion
| Supported pipeline types | metrics |
```

This receiver only produces metrics.  It is expected to be used in combination with the `awsemf` exporter, which exports metrics to CloudWatch Logs in the Embedded Metrics Format, which is why there are references to emitting "performance log events".  The only OTel telemetry signal it produces are metrics.